### PR TITLE
Expose all classes as exports at top level

### PR DESCRIPTION
**Pull request by marcinlichwala**
Exposed all classes as exports at top level using jscodeshift codemods.

### CODE REVIEW

**Comment by NicolasRannou** [comment]
@marcinlichwala thanks for the PR - 

Did you automatically update files using the `codemods` scripts?

File: src/ami.js [hunk 1]
```diff
@@ -1,29 +1,15 @@
...
-import Core from './core/core';
-import Geometries from './geometries/geometries';
-import Helpers from './helpers/helpers';
-import Loaders from './loaders/loaders';
-import Models from './models/models';
-import Parsers from './parsers/parsers';
-import Presets from './presets/presets';
-import Shaders from './shaders/shaders';
-import Widgets from './widgets/widgets';
+export * from "./cameras/cameras"
```
Comment (NicolasRannou) [inline src/ami.js]: Please use single quotes! :)

**Comment by marcinlichwala** [comment]
@NicolasRannou Yes I did and I can see only now that it doesn't always comply with the code styling. 

Will update to adjust, sorry :)

**Comment by marcinlichwala** [comment]
Also, Please let me know if you want the codemods in the repo as well because probably they won't be needed in the future after they are applied.

**Comment by NicolasRannou** [comment]
You are correct, let's take it out too

**Comment by NicolasRannou** [comment]
Thanks -  I'll go over it and merge it ASAP (currently traveling)

**Comment by NicolasRannou** [comment]
Related Hackfest issue: https://github.com/FNNDSC/ami/issues/183

**Comment by NicolasRannou** [comment]
Closes #183 
Thanks!

**Pull request merged**